Update wcag-ruleset-review-process.md (#486)
w3c/wcag-act: wcag-ruleset-review-process.md

Organizations and individuals, **rule providers**, can submit rules to be added to the WCAG Ruleset. Submitted rules must conform to the ACT Rules Format 1.0 specification and be accompanied by real-life implementations (specific criteria detailed later in this document). **Rule providers are expected to only contribute complete rules, as accurate as the rule providers can possibly make them.**

## Rule Review

> When proposing a new rule, or an update to a rule, there are four steps to go through:

Reviewer: Change "four steps" to "three steps".

> When proposing a new rule, or an update to a rule, there are four steps to go through:

Author: ```suggestion
When proposing a new rule, or an update to a rule, there are three steps to go through:
```

2. License of the rule
3. Information about any [implementations](#implementations)

### Surveying

A facilitator of the ACT Task Force creates a survey for the rule, in which Task Force members are asked the following questions:

- Does the rule follow the ACT Rules Format 1.0?
- Is the rule up to date?
- Are the assumptions acceptable?
- Is the implementation data correct?

> - Is the rule consistent with existing WCAG documents?

Author: ```suggestion
- Is the rule consistent with existing accessibility standards?
```

### Approving

When a rule passes the review stage, it is included in a draft version of the WCAG Ruleset. Depending on the types of changes and the availability of the Accessibility Guidelines (AG) Working Group, the ACT Task Force decides on a timeline for releasing updates to the WCAG Ruleset. To update the WCAG Ruleset, the ACT Task Force requests approval from the AG Working Group. The AG Working Group will have its own process for deciding whether or not to publish the next version of the WCAG Ruleset. For example, AG may decide to have Call For Consensus e-mail or send out a survey.

## Minor Updates

> Once a rule has been included in the (draft) WCAG ruleset, the rule provider can submit minor changes in a pull request directly to the draft. The ACT Task Force must decide on a call whether the changes need to be surveyed, following the [rule review process](#rule-review), or if it can be merged directly into the draft ruleset. Minor updates require a 48 hour call for consensus, except if the update is purely editorial.

Reviewer: I'm a little confused by this section. Later sections, Public Feedback and Annual Review, also talk about doing updates. However, this section only refers to "(draft) WCAG ruleset" rather than published rules. Is this section only related to the initial publication process? If so, then I think this needs to be clarified.

> Once a rule has been included in the (draft) WCAG ruleset, the rule provider can submit minor changes in a pull request directly to the draft. The ACT Task Force must decide on a call whether the changes need to be surveyed, following the [rule review process](#rule-review), or if it can be merged directly into the draft ruleset. Minor updates require a 48 hour call for consensus, except if the update is purely editorial.

Author: ```suggestion
Once a rule has approved by the ACT Task Force, the rule provider can submit minor changes in a pull request directly to the draft. The ACT Task Force must decide on a call whether the changes need to be surveyed, following the [rule review process](#rule-review), or if it can be merged directly into the draft ruleset. Minor updates require a 48 hour call for consensus, except if the update is purely editorial.
```

When a rule passes the review stage, it is included in a draft version of the WCAG Ruleset. Depending on the types of changes and the availability of the Accessibility Guidelines (AG) Working Group, the ACT Task Force decides on a timeline for releasing updates to the WCAG Ruleset. To update the WCAG Ruleset, the ACT Task Force requests approval from the AG Working Group. The AG Working Group will have its own process for deciding whether or not to publish the next version of the WCAG Ruleset. For example, AG may decide to have Call For Concensus e-mail or send out a survey.

## Minor Updates

> Once a rule has been included in the (draft) WCAG ruleset, the rule provider can submit minor changes in a pull request directly to the draft. The ACT Task Force must decide on a call whether the changes need to be surveyed, following the [rule review process](#rule-review), or if it can be merged directly into the draft ruleset. This requires a 48 hour call for concensus, except if the update is purely editorial.

Author: ```suggestion
Once a rule has been included in the (draft) WCAG ruleset, the rule provider can submit minor changes in a pull request directly to the draft. The ACT Task Force must decide on a call whether the changes need to be surveyed, following the [rule review process](#rule-review), or if it can be merged directly into the draft ruleset. Minor updates requires a 48 hour call for consensus, except if the update is purely editorial.
```

Reviewer 2: "Minor updates require", not "requires"

Since technologies and standards frequently change, rules require regular maintenance. To ensure quality over time, the ACT Task Force will review every rule in the WCAG Ruleset at least once a year, to ensure it is still current and correct. If any changes are deemed necessary, the ACT Task Force proposes an update to the rule to the original rule provider. If the change is taken up without further updates, the update is accepted without review.

> Every time a rule is surveyed, it is checked to make sure it is up to date. This survey counts as an annual review, so that only rules that have not been surveyed in 12 months need to go through a separate review. If a rule has not been reviewed for 12 months, an ACT facilitator opens an issue for it, and assigns it to that rule's liaison. The liason is has 10 work days to review if the rule needs to be updated. If the rule needs to be updated, the liaison has another 10 work days to propose any necessary changes to the rule provider.

Reviewer 3: ```suggestion
Every time a rule is surveyed, it is checked to make sure it is up to date. This survey counts as an annual review, so that only rules that have not been surveyed in 12 months need to go through a separate review. If a rule has not been reviewed for 12 months, an ACT facilitator opens an issue for it, and assigns it to that rule's liaison. The liason has 10 work days to review if the rule needs to be updated. If the rule needs to be updated, the liaison has another 10 work days to propose any necessary changes to the rule provider.
```
